wordpress-mobile/WordPress-Android
# [Dependabot] Move Included Builds to Fix Dependabot

This is a replicate fix of [WCAndroid's PR](https://github.com/woocommerce/woocommerce-android/pull/7769). Please refer to that for additional information.

-----

This change moves the configuration of included builds to a separate file in order to fix Dependabot as it doesn't understand the content of, for example `ext.localFluxCPath`. Dependabot looks for known schemas of version declaration on both, the 'build.gradle' and 'settings.gradle' file.

As such moving this included builds specific configuration to a separate file, with a different naming convention, will make Dependabot ignore this file. As such, it will start resolving the dependencies again.

FYI: See Dependabot error below:

```
Dependabot couldn't find a build.gradle.

Dependabot requires a build.gradle to evaluate your
Java dependencies. It had expected to find one at the path:
/ext.localFluxCPath/build.gradle.

If this isn't a Java project, you may wish to disable updates
for it in the .github/dependabot.yml config file in this repo.
```

-----

To test:

Try to include/exclude builds from your `local-builds.gradle` and verify everything is working as expected.

-----

## Regression Notes

1. Potential unintended areas of impact

N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)

To `To test` section above.

3. What automated tests I added (or what prevented me from doing so)

N/A

-----

PR submission checklist:
